packages/space-opera/src/components/camera_settings/components/fov_limits.ts

```diff
@@ -75,7 +60,8 @@ export class FovLimits extends LimitsBase {
   }
 
   get currentPreviewValue() {
-    return Math.round(this.currentCamera?.fieldOfViewDeg ?? DEFAULT_MIN_FOV);
+    const currentCamera = getCameraState(getModelViewer()!);
```

Comment:
getModelViewer returns document.querySelector('model-viewer-preview')?.modelViewer

this was added because we removed modelViewer and currentCamera from the state